.pre-commit-config.yaml

```diff
@@ -49,6 +49,7 @@ repos:
     rev: "0.6.6"
     hooks:
       - id: pydoclint
+        exclude: ^python/ray/serve/tests/test_config_files/syntax_error\.py$
```

Comment:
This should probably be in the `--exclude` line further down instead of in pre-commit directly?

Or at least in both :D

Reply:
Good point! Updated

<img width="901" alt="image" src="https://github.com/user-attachments/assets/a1b29519-0f9b-42c8-80e7-668df8578e3c" />